src/editor/components/Setting/ComponentEvent/index.tsx

```
@@ -1,201 +1,88 @@
-  /**
-   * @description 删除指定事件下的一个动作。
-   */
-  function deleteAction(event: ComponentEvent, index: number) {
-    if (!curComponent) {
-      return;
-    }
+  // useEventSearch Hook 负责事件列表的搜索、评分和排序
+  const allEvents = curComponent
+    ? componentConfig[curComponent.name]?.events
+    : [];
+  const filteredEvents = useEventSearch(allEvents, searchKeyword);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)